packages/lib/shared/utils/error-filters.ts

```diff
@@ -57,7 +57,7 @@ export function isUnbalancedAddError(error?: Error | null): boolean {
   return false
 }
 
-export function isUnbalancedAddErrorMessage(error: Error | null): boolean {
+export function isUnbalancedAddErrorMessage(error?: Error | null): boolean {
```

Comment:
why is `error` optional?

Reply:
We call it with` isUnbalancedAddErrorMessage(priceImpactQuery.error)` where error can be undefined.

Reply:
looks like it's defined or null everywhere